samples/bookinfo/src/productpage/uid_entrypoint

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+if ! whoami &> /dev/null; then
```

Comment:
Looks like the `&> /dev/null` doesn't do what you'd expect, because the script is running under `sh` instead of `bash`.

Reply:
This is from the Openshift docs :)